modules/docs/mdx/9.0-UPGRADE-GUIDE.mdx

```diff
@@ -38,6 +38,38 @@ any questions.
 
 ## Codemod
 
+A codemod is a script that makes programatic transformations on your codebase by traversing the AST,
```

Comment:
Possibly add a link here for AST? 

`[AST](https://www.codeshiftcommunity.com/docs/understanding-asts)`

I found this link on codeshift community which seems pretty good but could use others.